docker-compose/compose.yaml

```diff
@@ -84,8 +80,15 @@ services:
         http://shared-file-store:8099/alfresco/api/-default-/private/sfs/versions/1/file
     ports:
       - "8095:8095"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8095/actuator/health"]
+      interval: 60s
+      timeout: 10s
```

Comment:
for simple curl it's reasonable to stay low even `2s`

be consistent across different services